roadmap/README.md

```diff
@@ -27,6 +27,7 @@ _Status description:_
 | ✔️| Rename switch state `default` to `defaultCondition` to avoid keyword conflicts for SDK's | [spec doc](../specification.md) |
 | ✔️| Add description of additional properties | [spec doc](../specification.md) |
 | ✔️| Rename Parallel `completionType` values | [spec doc](../specification.md) |
+| ✔️| Rename `workflowId` from ParallelState (use subFlow action instead) | [spec doc](../specification.md) |
```

Comment:
"Remove" - I'll fix in a separate pr cause I think this needs to go in with us not having a meet for a while